### ISSUE TITLE
Parser deletes the first element of a node's body when defining node attributes

### ISSUE THREAD

**Issue by eudoxia0** [Shinmera/plump-tex#7]
So here's a weird bug:

```lisp
CL-USER> (plump:serialize (plump-tex:parse "\\a{\\b[key=value]{test}}"))
<a><b key="value">est</b></a>
```

**Comment by Shinmera** [comment]
The fuck. I thought I had fixed that.

**Comment by Shinmera** [comment]
I *did* fix that! See #6.

Update your sources before you report bugs, man.

**Comment by eudoxia0** [comment]
oops, sorry for not checking the issue list. Thanks!
